php_02/data.txt

```diff
@@ -0,0 +1,21 @@
+Име на курса:dddddd
```

Comment:
It is not a good idea to commit this kind of information. In the real world this information would have been stored in a database and not in txt file. User data is not meant to be committed in version control systems (like github) but in some kind of data storage. Please delete the file.